app/main/Mindfulness.tsx

```diff
@@ -3,39 +3,32 @@
 "use client";
 
 import '../css/Mindfulness.css'; // Import CSS file
+import { videoList } from '@/lib/data'; // Import video data
 
 interface VideoPlayerProps {
   videoId: string;
-  Label: string;
+  label: string;
 }
 
-const generateVideoPlayer = ({ videoId, Label }: VideoPlayerProps) => (
-  <div className="video-wrapper">
+const generateVideoPlayer = ({ videoId, label }: VideoPlayerProps) => (
+  <div className="video-wrapper" key={label}>
     <iframe
       className='video'
       title='Youtube player'
-      aria-label={Label}
+      aria-label={label}
       src={`https://www.youtube.com/embed/${videoId}`}
     ></iframe>
   </div>
 );
 
 function Mindfullness() {
```

Comment:
Would it be clearer to name this something like `MindfullnessTabContents`?

Reply:
I updated the aria-label to be alt-text for the video.